src/helpers/steg/barn.ts

```diff
@@ -5,6 +5,7 @@ import { dagensDato, formatIsoDate } from '../../utils/dato';
 import { hentUid } from '../../utils/uuid';
 import { EBarn, IBarn } from '../../models/barn';
 import { ESvar } from '../../models/spørsmålogsvar';
+import navfaker from 'nav-faker';
```

Comment:
Føles nesten litt skummelt att ta inn noe som heter navfaker i prod-kode :P

Reply:
Enig - todelt ang den der. 
Den kan jo DNr-logikk som jeg helst vil slippe å lage selv, men den kan også generere tvilsomme data :D

Reply:
Hvis det finnes en annen `nav`-pakke for dnr-logikk så rop ut!

Reply:
De får ta og splitte repot :trollface:

Reply:
You fix? ;)